meson_options.txt

```diff
@@ -2,7 +2,7 @@ option('cli', type: 'boolean', value: true,
     description: 'Build the CLI')
 option('experimental', type: 'boolean', value: true, yield: true,
```

Comment:
Do we still need the yield here?

Reply:
Yes. This one is actually valuable. Keeps experimental in check between hse-python and hse. The tests yield was just me not understanding the best time to use the feature.